misc/kubernetes-examples/blog1/mysql-deployment-vsphere.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
```

Comment:
How about locking the branch and using its reference rather than merging to vDVS master? It leads to confusion in the future keep adding non vDVS stuff to vDVS/misc folder.